lib/ansible/modules/commands/command.py

```diff
@@ -74,7 +74,8 @@
   register: mymotd
 
 - name: Run the command if the specified file does not exist.
-  command: /usr/bin/make_database.sh arg1 arg2 creates=/path/to/database
+  command: /usr/bin/make_database.sh arg1 arg2
+  creates: /path/to/database
```

Comment:
this is incorrect syntax

Reply:
It passes ansible-syntax-check?
<code>
---
- hosts: all 
  tasks:
      - name: Run the command if the specified file does not exist.
        command: /usr/bin/make_database.sh arg1 arg2
        creates: /path/to/database
</code>

##### ANSIBLE VERSION
ansible 2.4.2.0
  config file = /Users/jsmith39/src/wvu/ansible/ansible.cfg
  configured module search path = [u'/Users/jsmith39/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.14 (default, Jan  9 2018, 14:55:52) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]

Reply:
Update -  I do see that the syntax check does print a warning about an unknown attribute now.